docs/1___overview.adoc

```diff
@@ -154,12 +154,12 @@ This major release of the SSP standard provides support for FMI 3.0, as well as
 Even though SSP 2.0 is a major version release, all SSP 1.0 conforming artifacts are considered to be fully conforming to SSP 2.0.
 SSP 2.0 conforming artifacts are only compliant to the SSP 1.0 standard if they use a 1.0 version number and only use features available in SSP 1.0.
 
-The following changes were performed as part of the 2.0.0 major release:
+An overview of the changes in the 2.0.0 major release are provided in the bulleted lists below. Many different advancements to the specification have been discussed; however, the SSP 2.0 specification has, in its current status, been deduced as a feasible least-common denominator for the exchange of simulation packages that is able to provide value for both end-users and tool suppliers.
```

Comment:
We should avoid rationale or forward looking statements here: This is the description of the set of changes the standard contains, the reason why it contains these and not others, and what it might contain in the future should be immaterial to describing the current state of affairs.

If we want to explain ourselves, we should do that as part of the press release, but not in the main document.